editor/utils/test/dom.js

```diff
@@ -1,35 +1,30 @@
 /**
  * External dependencies
  */
-import { expect } from 'chai';
-
-/**
- * Internal dependencies
- */
 import { isEditableElement } from '../dom';
```

Comment:
This section should now be titled `Internal dependencies`, not `External dependencies`.